contracts/Pod.sol

```diff
@@ -536,17 +539,22 @@ contract Pod is
         internal
         returns (uint256)
     {
+        uint256 balanceBefore = token.balanceOf(address(this));
+
         // Withdraw from Prize Pool
-        uint256 exitFeePaid =
-            _prizePool.withdrawInstantlyFrom(
-                address(this),
-                amount,
-                address(ticket),
-                maxFee
-            );
-
-        // Exact Exit Fee
-        return exitFeePaid;
+        _prizePool.withdrawInstantlyFrom(
+            address(this),
+            amount,
+            address(ticket),
+            maxFee
+        );
+
+        uint256 balanceAfter = token.balanceOf(address(this));
```

Comment:
Ah!  There is an optimization here.  Assign `token` to a local var so that it doesn't do two SLOADs